.github/workflows/gradle.yml

```diff
@@ -16,10 +20,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 15
+    - name: Set up JDK 8
```

Comment:
Is it made to allow usage of the library with java 8+?

Reply:
Yes, it is

Reply:
I decided to expand our potential users.
I'm assuming users of Java 8 more than users of Java 8+